src/SynchroFeed.Library/Model/Package.cs

```diff
@@ -382,7 +382,7 @@ public bool Equals(Package package)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("ReSharper", "NonReadonlyMemberInGetHashCode")]
         public override int GetHashCode()
         {
-            return $"{this.Id.ToLower()}{this.Version}".GetHashCode();
+            return this.ToString().ToLower().GetHashCode();
```

Comment:
I figured with the ToString now leveraging the version, this could be simplified a little.